Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/index.md

```diff
@@ -61,12 +61,17 @@ By default, the following Field Types are available:
 - **Single Choice**:
     Displays a list of items with a radio button for each item where the user can select a single option.
 
-    ![singlechoice](images/singlechoice-v9.png)
+    ![Single choice](images/singlechoice-v9.png)
 
 - **Title and Description**:
-    Displays a title and description that are set as default values.
+    Displays a read-only title and description for a set of form fields.
```

Comment:
```suggestion
    Displays a read-only title and description for a set of Form fields.
```

Reply:
I'd be minded to keep this as "form" as here we are talking about "a form" rather than "Umbraco Forms" the product.  But up to you.

Reply:
Makes sense.